src/Chapter1.hs

```diff
@@ -640,8 +665,11 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n
+    | numberCheck == 0 = n
+    | otherwise = firstDigit numberCheck
+    where numberCheck = div n 10
```

Comment:
It looks like this function hangs on negative numbers :thinking: 

See how `div` works for them:

```haskell
ghci> div (-1) 10
-1
```

Reply:
Thanks for the review! I pushed up some revisions and for the last function (firstDigit) I wasn't sure if the intent of the function was to return a negative first digit if a negative integer was passed, so I included a commented version for doing just that. Otherwise, it just returns the absolute first digit.